curriculum/challenges/english/25-front-end-development/quiz-css-positioning/66ed8ffcf45ce3ece4053eb5.md

```diff
@@ -17,439 +17,439 @@ Answer all of the questions below correctly to pass the quiz.
 
 #### --text--
 
-Placeholder question
+The main purpose of the float property in CSS is:
 
 #### --distractors--
 
-Placeholder distractor 1
+To create rounded corners
 
 ---
 
-Placeholder distractor 2
+opacity
 
 ---
 
-Placeholder distractor 3
+To center vertically
 
 #### --answer--
 
-Placeholder answer
+The property is used to align elements to either the left or the right of their container.
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which CSS property is used to set the stack order of an element?
 
 #### --distractors--
 
-Placeholder distractor 1
+position
 
 ---
 
-Placeholder distractor 2
+bg-green
 
 ---
 
-Placeholder distractor 3
+float
 
 #### --answer--
 
-Placeholder answer
+z-index
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What happens when you apply the property position: relative to an element in CSS ?
 
 #### --distractors--
 
-Placeholder distractor 1
+It removes the element from document flow
 
 ---
 
-Placeholder distractor 2
+It only moves the element relative to the viewport
 
 ---
 
-Placeholder distractor 3
+This will keep the element in a fixed position relative to the viewport.
 
 #### --answer--
 
-Placeholder answer
+It moves an element relative based on its normal position
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which CSS property would you use to fix an element at a certain position on the page so that it does not move when scrolling occurs?
 
 #### --distractors--
 
-Placeholder distractor 1
+position: absolute
```

Comment:
```suggestion
`position: absolute`
```

backticks are needed when writing code, please add to the other distractors and answers that are missing them

Reply:
Sure!